java/core/src/main/java/sleeper/core/properties/instance/TableStateProperty.java

```diff
@@ -176,6 +176,11 @@ private static UserDefinedInstancePropertyImpl.Builder propertyBuilder(String pr
             .description("The maximum given concurrency allowed for the transaction deletion lambda.\n" +
                     "See maximum concurrency overview at: https://aws.amazon.com/blogs/compute/introducing-maximum-concurrency-of-aws-lambda-functions-when-using-amazon-sqs-as-an-event-source/")
             .propertyGroup(InstancePropertyGroup.TABLE_STATE).build();
+    UserDefinedInstanceProperty TRANSACTION_DELETION_LAMBDA_TIMEOUT = Index.propertyBuilder("sleeper.statestore.transactionlog.deletion.lambda.timeout.seconds")
```

Comment:
The names of the other properties that configure the transaction deletion lambda all start with `sleeper.statestore.transactionlog.transaction.deletion`. Please can we make them match?

I'm not sure if it's worth having both `.transactionlog` and `.transaction`, but it seems important to keep them all the same.